package.json

```diff
@@ -46,6 +46,7 @@
   "dependencies": {
     "@subql/types-cosmos": "^3.2.3",
     "@types/node": "^17.0.21",
+    "cross-fetch": "^4.0.0",
     "pino": "^7.8.0",
```

Comment:
Why not use `axios`? 

After comparing axios and cross-fetch, I think axios has more features compared to cross-fetch. It's easier to use, has automatic JSON parsing, and more popular :D 

cross-fetch does have the advantage of being lightweight at `9.8kB` minified, compared to axios's `30.5kB`.

Reply:
axios unfortunately does not work in the restricted environment properly. i've tried a node-fetch as well which did not work

only cross-fetch seems to work properly